.rubocop.yml

```diff
@@ -44,10 +44,14 @@ Layout/SpaceInsideBrackets:
   Enabled: false
 Lint/EndAlignment:
   Severity: error
+Lint/RescueWithoutErrorClass:
+  Enabled: false
 Lint/UnreachableCode:
   Severity: error
 Lint/UselessAccessModifier:
   Enabled: false
+Lint/Void:
+  Enabled: false
```

Comment:
What even is this? :joy:

Reply:
Oh we must return some value and it doesn't like that! We can enable it in a subsequent pass.